openshift/templates/issuer-web/config/openvp-candy/dev/config.json

```diff
@@ -0,0 +1,43 @@
+{
+  "env": "development",
+  "issuer": {
+    "name": "Unverified Person Issuer"
+  },
+  "help": {
+    "enabled": false,
+    "displayOnFirstVisit": false
+  },
+  "inviteRequired": false,
+  "authentication": {
+    "enabled": false,
+    "autoSignOut": true,
+    "oidcSettings": {
+      "authority": "https://toip-vc-authn-controller-dev.apps.silver.devops.gov.bc.ca",
+      "clientId": "openvp-dev",
+      "redirectUri": "https://openvp-issuer-dev.apps.silver.devops.gov.bc.ca/oidc-callback",
```

Comment:
URLs in configuration files need to be updated with the appropriate suffixes (in this case, `openvp-candy`).

Applies to other `config.json` files as well.